### PR TITLE
Add Perform Component to Storybook

### PR DESCRIPTION
**Pull request by colefriedlaender**
Perform Component

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
LGTM 💯

File: components/Perform/Perform.module.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.container {
+  background-color: var(--text-button-color);
+  height: 107px;
```
Comment (phlgr) [inline components/Perform/Perform.module.css]: Why are you using these numbers?

File: components/Perform/Perform.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  padding-right: 20px;
+  box-shadow: 0 2px 5px;
+}
+.head {
+  padding-top: 10px;
+  text-align: center;
+}
+.perform {
+  color: var(--main-color);
+  font-size: 15px;
```
Comment (phlgr) [inline components/Perform/Perform.module.css]: Set the font-size via `em`

File: components/Perform/Perform.module.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  text-align: center;
+}
+.performRate {
+  font-size: 15px;
```
Comment (phlgr) [inline components/Perform/Perform.module.css]: Same as the above

**Review by phlgr — CHANGES_REQUESTED** [review]

**Pull request merged**